test/verify/check-journal

```diff
@@ -427,26 +427,24 @@ s.send(b"PRIORITY=3\\nFOO=bar\\n")'
 
         self.login_and_go("/system/logs")
 
-        m.execute("ulimit -c unlimited; python3 -c 'import os; os.kill(os.getpid(), 11)' || true")
+        m.execute("ulimit -c unlimited; timeout --signal=SEGV 0.1s sleep 5s || true")
 
         sel = "#journal-box .cockpit-logline .cockpit-log-message:contains('crashed in %s')" % self.crash_fn
         b.click(sel)
 
-        b.wait_in_text("#journal-entry-id", "python")
+        b.wait_in_text("#journal-entry-heading", "sleep")
         sel = "#journal-entry-fields .nav .btn-danger"
         b.click(sel)
 
-        b.wait_visible("#journal-box")
         b.wait_in_text('#journal-box', "crashed in " + self.crash_fn)
 
         sel = "#journal-box .cockpit-logline .cockpit-log-message:contains('crashed in %s')" % self.crash_fn
         b.click(sel)
 
-        b.wait_in_text("#journal-entry-id", "python")
+        b.wait_in_text("#journal-entry-heading", "sleep")
         # details view should hide log view
         b.wait_not_visible('.cockpit-log-panel')
-        b.wait_present("#journal-entry-message:contains('crashed in %s')" % self.crash_fn)
```

Comment:
Why was that removed? Can't abrt figure out the function it crashed in for the sleep command? If not, then moving from python to timeout+sleep wasn't a good move.

Reply:
I don’t really recall by now, but I think it can be added back.

Reply:
The ABRT argument is completely orthogonal to the issue. I had simply removed the element previously and forgot to add this line back when I added the element back.

Reply:
A bit off-topic for this, but also see the naughties for ABRT.